site/_components/featured-story.njk

```diff
@@ -0,0 +1,39 @@
+<!--{#
+  Renders a featured story component.
+
+  Arguments:
+    @param {object} data - Object containing the following attributes.
+    @param {string} data.eyebrow - Eyebrow for the component.
+    @param {string} data.title - Title for the component.
+    @param {string} data.desc - Copy for the component.
+    @param {object} data.cta - Object containing the cta attributes.
+    @param {string} data.cta.url - Url for the cta.
+    @param {string} data.cta.text - Text for the cta. 
+    @param {string} data.image: String for the image path.
+    @param {string} data.alt - String for the image alt.
+
+    @param {object} modifiers - Object containing style variations for the component.
+    @param {string} modifiers.custom Class that will allow to override CSS properties for this component.
+        Use this modifier if you want to override --overlay-color and/or --featured-shape-color.
+
+ #}-->
+{% macro featuredStory(data, modifiers) -%}
+<div class="featured-story">
+  {% for image in data.images %}
+  <img class="featured-story__image" src="{{image.image}}" alt="{{image.alt}}" />
+  {% endfor %}
+
+  <div class="wrapper--padded featured-story__wrapper">
+    <article class="featured-story__container {{modifiers.custom}}">
+      <section class="featured-story__copy-wrapper">
+        <div class="featured-story__logo">
+          <img class="featured-story__logo--image" src="{{data.app.logo}}" alt="{{data.app.alt}}" />
```

Comment:
```suggestion
          <img class="featured-story__logo--image" src="{{data.app.logo}}" alt="{{data.app.name}}, {{data.app.company}}" />
```

We don't have alt text supplied for the logos because we do `aria-hidden` on story pages in favor of displaying the app name and company. In lieu of displaying that, the alt text should be the app name and company.